llama/runner/runner.go

```diff
@@ -211,6 +218,7 @@ type Server struct {
 	// required for image embeddings
 	image *ImageContext
 
+	// TODO (jmorganca): make this n_batch
```

Comment:
Did you mean to add this (back?)

Reply:
I just moved it - based on some of the changes that I made, it's not as obviously applicable in the original location. I wasn't try to change anything related to it in the patch series.